ci-operator/config/openshift/lightspeed-operator/openshift-lightspeed-operator-main.yaml

```diff
@@ -3,6 +3,18 @@ base_images:
     name: golangci-lint
     namespace: ci
     tag: v1.54.2
+  lightspeed-console-plugin:
+    name: lightspeed-console-plugin
+    namespace: openshift
```

Comment:
i think these need to be "ols", as that is where we are promoting to, not openshift